cli/src/prepare.js

```diff
@@ -6,7 +6,7 @@ import fs from 'node:fs/promises';
 import os from 'node:os';
 import path from 'node:path';
 
-const SKIP = ['.git', '.gitignore'];
+const SKIP = ['.git'];
```

Comment:
I think originally because we wanted to use the `.gitignore` from the blueprint -- but what you've done is probably the more correct thing to do (after I've converted a few more packages without the gitignore, I've run in to a number of manual updates related to _not_ having your change)